tools/topology/topology2/include/components/cavs/src.conf

```diff
@@ -0,0 +1,96 @@
+#
+#
+# A generic src component. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.src.N.attribute_name"
+#
+# Usage: this component can be used by instantiating it in the parent object. i.e.
+#
+# 	Object.Widget.src."N" {
+#		period_sink_count	2
+#		period_source_count	2
+#		format			"s24le"
+#		rate_out		48000
```

Comment:
can you describe input rates? is 48k the only supported option? what about s32le?

Reply:
I will update PR to include all supported input rate